server/repositories/offender.js

```diff
@@ -74,6 +90,10 @@ function offenderRepository(httpClient) {
     return httpClient.get(`${endpoint}?${query.join('&')}`);
   }
 
+  function getPrisonDetailsFor(prisonId) {
+    return httpClient.get(`${getBaseUrlFrom(config)}/agencies/${[prisonId]}`);
+  }
+
```

Comment:
This looks like it's inconsistent with the config-tree approach for generating URLs (hence why it relies on a _new_ `base` variant). That's maybe not the end of the world because we may want to rethink that approach (see another comment in this PR) – I actually favour this approach because it's clearer what your API call is than the other examples.

Reply:
This has been removed, and for the refactored code we have decided to avoid centralising API paths in the config in favour of picking up the environment variable directly in the constructor (see PrisonApiRepository)